gulpfile.js

```diff
@@ -6,7 +6,7 @@ const gulp = require('gulp');
 process.env.NODE_ENV = process.env.NODE_ENV || 'development';
 
 require('godaddy-test-tools')(gulp, {
-  lint: { files: ['*.js', 'lib/**/*.js', 'test/*.js'] },
+  lint: { files: ['*.js', 'lib/**/*.js', 'test/**/*.js'] },
```

Comment:
I'm not super familiar with gulp, but depending on platform and how this gets used you may need to include both `test/*.js` and `test/**/*.js`.